scalyr_agent/agent_main.py

```diff
@@ -1814,8 +1860,8 @@ def __calculate_overall_stats(
 
         return result
 
-    def __report_status_to_file(self):
-        # type: () -> str
+    def __report_status_to_file(self, health_check=False):
```

Comment:
I don't think you really need to pass in `health_check` anymore, right?  The format should already be set to JSON and so we should just return back the normal JSON status object.